### PR TITLE
Implement looping integer list view

### PR DESCRIPTION
**Pull request by mathieupost**
Solves https://github.com/MarcinusX/NumberPicker/issues/11

### CODE REVIEW

File: lib/numberpicker.dart [hunk 1]
```diff
@@ -116,13 +121,23 @@ class NumberPicker extends StatelessWidget {
   /// if min=0, max=5, step=3, then items will be 0 and 3.
   final int step;
 
+  ///Repeat values infinitely
+  final bool infiniteLoop;
+
+  ///Amount of items
+  int itemCount;
```
Comment (MarcinusX) [inline lib/numberpicker.dart]: StatelessWidget should not have mutable field. Could you move it back as a variable in method?

**Comment by mathieupost** [comment]
Sure! I'll take a look probably tomorrow ;)

**Comment by MarcinusX** [comment]
Just noticed that on iOS, overscroll goes out of bound, when you scroll over 0, it gets to 10 for a while :(

**Comment by mathieupost** [comment]
Ah I've only tested on Android, but I'll check out the issue on iOS.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by mathieupost** [comment]
Fixed ;)

**Comment by MarcinusX** [comment]
Thank you for your contribution and patience!

**Pull request merged**